vpr/src/place/initial_placement.cpp

```diff
@@ -21,12 +31,20 @@ static int check_macro_can_be_placed(t_pl_macro pl_macro, int itype, t_pl_loc he
 static int try_place_macro(int itype, int ipos, int isub_tile, t_pl_macro pl_macro);
 static void initial_placement_pl_macros(int macros_max_num_tries, std::vector<std::vector<int>>& free_locations);
 
-static void initial_placement_blocks(std::vector<std::vector<int>>& free_locations, enum e_pad_loc_type pad_loc_type);
+static void initial_placement_blocks(std::vector<std::vector<int>>& free_locations, enum e_pad_loc_type pad_loc_type, std::vector<ClusterBlockId> sorted_blocks);
 
 static t_physical_tile_type_ptr pick_placement_type(t_logical_block_type_ptr logical_block,
                                                     int num_needed_types,
                                                     std::vector<std::vector<int>>& free_locations);
 
+//Assign scores to each block based on macro size, floorplanning constraints, and number of equivalent tiles
```

Comment:
What is it for -> relative placement. Higher score = more difficult to place, is a priority during initial placement